Sources/Core/UI/Viewers/AV/AVViewController.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  AVViewController.swift
+//  BoxPreviewSDK-iOS
+//
+//  Created by Sujay Garlanka on 12/19/19.
+//  Copyright © 2019 Box. All rights reserved.
+//
+
+import BoxSDK
+import Foundation
+import AVKit
+
+public class AVViewController: UIViewController, PreviewItemChildViewController {
+
+    // MARK: - Properties
+    
+    weak var fullScreenDelegate: PreviewItemFullScreenDelegate?
+    var toolbarButtons: [UIBarButtonItem] = []
+    
+    private var url: URL
+    private var file: File?
+    private var AVPlayerVC: AVPlayerViewController
+    private var client: BoxClient?
+    
+    private lazy var titleView: CustomTitleView = {
+        let view: CustomTitleView = CustomTitleView()
+        return view
+    }()
+    
+    // MARK: - Initializer
+    
+    public init(url: URL, file: File? = nil, client: BoxClient? = nil, actions: [FileInteractions]) {
+        self.url = url
+        self.file = file
+        self.AVPlayerVC = AVPlayerViewController()
+        self.client = client
+        super.init(nibName: nil, bundle: nil)
+        // Not showing controls, so when video players doesn't flash a play button before it automatically starts playing
+        self.AVPlayerVC.showsPlaybackControls = false
+        self.setupPlayer()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
```suggestion
        super.init(coder: coder)
```
Is this better here?

Reply:
Copied this over from the other view controllers like ImageViewController.swift.